# This message holds all joints params
#
# Arrays should have same size

string[] names
TypeJointParams[] params